src/components/LoadDatasetButton.tsx

```diff
@@ -313,12 +314,7 @@ export default function LoadDatasetButton(props: LoadDatasetButtonProps): ReactE
               </Space.Compact>
             </div>
           </div>
-
-          {errorText && (
-            <p>
-              <span style={{ color: theme.color.text.error }}>{errorText}</span>
-            </p>
-          )}
+          {errorText && <MessageCard type="error">{errorText}</MessageCard>}
```

Comment:
Use the new MessageCard component for load error messaging in the load modal

![image](https://github.com/user-attachments/assets/7a257e3d-eb37-445d-83f9-2ae50ce7bbf7)